src/js/content/scanForCSS.ts

```diff
@@ -63,11 +64,38 @@ function isValidCSSRule(rule: CSSRule): boolean {
     return false;
   }
 
-  if (!(rule instanceof CSSGroupingRule || rule instanceof CSSKeyframesRule)) {
+  if (
+    !(
+      rule instanceof CSSGroupingRule ||
+      rule instanceof CSSKeyframesRule ||
+      rule instanceof CSSImportRule
+    )
+  ) {
     return true;
   }
 
-  return [...rule.cssRules].every(isValidCSSRule);
+  let rulesToCheck: Array<CSSRule> = [];
+
+  if (rule instanceof CSSImportRule) {
+    const styleSheet = rule.styleSheet;
+    if (styleSheet != null) {
+      ensureCORSEnabledForStylesheet(styleSheet);
+      rulesToCheck = [...styleSheet.cssRules];
```

Comment:
If an error is thrown inside ensureCORSEnabledForStylesheet() due to CORS, line 83 will still run causing an error. Maybe just short-circuit and return false?

Reply:
Should be fine since we have a few places today where we intentionally `invalidateAndThrow`

https://github.com/ezzak/meta-code-verify/blob/main/src/js/contentUtils.ts#L28